### PR TITLE
Issue #3104801 by chr.fritsch: Metatag test fails with 1.11 release 

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-4.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

**Comment by chrfritsch** [comment]
This was failing because of https://www.drupal.org/node/3099168

File: tests/src/FunctionalJavascript/ThunderMetaTagTrait.php [hunk 1]
```diff
@@ -44,7 +44,12 @@ public function checkMetaTags(array $metaTagConfiguration) {
     foreach ($metaTagConfiguration as $metaTagName => $value) {
       $metaTag = explode(' ', $metaTagName);
 
-      $this->checkMetaTag($metaTag[1], $value);
+      if ($metaTag[1] == 'title') {
+        $this->assertSession()->elementContains('xpath', '//title', $value);
```
Comment (mtodor) [inline tests/src/FunctionalJavascript/ThunderMetaTagTrait.php]: Can we move this logic inside `checkMetaTag`?

Comment (dbosen) [reply tests/src/FunctionalJavascript/ThunderMetaTagTrait.php]: We should test for title independent of the other meta information. This way we can have the sam,e test for all versions of metatag module  and do not have to increase the min version, just for our tests

Comment (chrfritsch) [reply tests/src/FunctionalJavascript/ThunderMetaTagTrait.php]: I wouldn't do that. We check the title tag now and not a meta tag anymore

**Review by dbosen — APPROVED** [review]

**Pull request merged**